modules/swagger-codegen/src/main/resources/objc/model-body.mustache

```diff
@@ -13,7 +13,11 @@
   }
   return self;
 }
-
+{{#vars}}{{^vendorExtensions.x-unique-id-key}}{{/vendorExtensions.x-unique-id-key}}{{#vendorExtensions.x-unique-id-key}}
+-(NSString*)indexPropertyName {
+    return @"{{name}}";
+}
+{{/vendorExtensions.x-unique-id-key}}{{/vars}}
```

Comment:
For Ruby on Rails, we need something similar to ensure a model property is unique, e.g.
```
validates_uniqueness_of :admin_user_id, :message => "This user has yet a competitors list"
```
Ref: http://stackoverflow.com/a/12764552/677735 

cc @zlx